app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -4,16 +4,12 @@ class PasswordConfirmController < BaseController
       self.required_step = 'password_confirm'
 
       def create
-        result, personal_key = Api::ProfileCreationForm.new(
-          password: verify_params[:password],
-          jwt: verify_params[:user_bundle_token],
-          user_session: user_session,
-          service_provider: current_sp,
-        ).submit
+        result, personal_key, gpo_code = form.submit
 
         if result.success?
           user = User.find_by(uuid: result.extra[:user_uuid])
           add_proofing_component(user)
+          session[:last_gpo_confirmation_code] = gpo_code if gpo_code
```

Comment:
Is this used?  I think ideally we don't store the GPO confirmation code in the session.

Reply:
This is just updating the new flow to match expected behavior that already existed previously.

Reply:
ohhh, I see, I missed it would be nil in the non-local case

Would it be too defensive to also put the `FeatureManagement.reveal_gpo_code?` guard around this assignment?

Reply:
> ohhh, I see, I missed it would be nil in the non-local case
> 
> Would it be too defensive to also put the `FeatureManagement.reveal_gpo_code?` guard around this assignment?

Yeah I'm open to that. I'll update.

Reply:
Updated  in 7db7fa218.